aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -292,7 +292,7 @@ static RestoreDbInstanceToPointInTimeRequest restoreDbInstanceToPointInTimeReque
             final ResourceModel model,
             final Tagging.TagSet tagSet
     ) {
-        final Instant restoreTimeInstant = StringUtils.isNotBlank(model.getRestoreTime()) ? ZonedDateTime.parse(model.getRestoreTime()).toInstant() : null;
+        final Instant restoreTimeInstant = StringUtils.isNotBlank(model.getRestoreTime())? Commons.parseTimestamp(model.getRestoreTime()) :null;
```

Comment:
I think we would benefit from consistent validation of timestamp across DBInstance and DBCluster

Reply:
Valiodation happens in early phase on request in handler here. 
```
        Validations.validateTimestamp(request.getDesiredResourceState().getRestoreTime());
```